CHANGELOG.md

```diff
@@ -83,6 +83,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Split out model tutorial into writing the model, and another tutorial for
   packaging the model.
 - IntegrationCLITestCase creates a new directory and chdir into it for each test
+- Update input directory definition in shouldi python example
```

Comment:
Since this was changed and fixed within this release we're going to skip the changelog entry

```suggestion
```